src/main/java/com/cricket/project/exception/PlayerException.java

```diff
@@ -0,0 +1,7 @@
+package com.cricket.project.exception;
+
+public class PlayerException extends Exception{
```

Comment:
Usually we create unchecked exceptions for these like RuntimeExceptions so that we don't need to add throws in every method signature

Reply:
okay noted.